examples/DayPickerRangeControllerWrapper.jsx

```diff
@@ -18,6 +18,10 @@ const propTypes = forbidExtraProps({
   autoFocusEndDate: PropTypes.bool,
   initialStartDate: momentPropTypes.momentObj,
   initialEndDate: momentPropTypes.momentObj,
+  range: PropTypes.shape({
+    before: PropTypes.func,
```

Comment:
This propType also allows an empty object; it seems like either one or both of these properties is required? If so, perhaps something like `or([justBefore, justAfter, bothBeforeAndAfter])` might be better?

Reply:
That's right, at least one is required, I can make an update here.